projectPage.css

```diff
@@ -1,57 +1,36 @@
-.vid {
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Oswald:wght@500&display=swap');
+
+
+/* HEADER 
+=================================================*/
+.header-vid {
     object-fit: cover;
-    width: 100vw;
```

Comment:
viewport width is dynamic however it's relative to the viewport, while % is relative to the container. in most cases you want the container. (exceptions might be full page modals, loading screens, etc.)